package.json

```diff
@@ -9,12 +9,13 @@
   "dependencies": {
     "autolinker.js": "0.7.0",
     "chartist": "^0.11.0",
+    "date-fns": "^1.28.5",
     "fastclick": "^1.0.6",
     "font-awesome": "4.7.0",
     "history": "4.6.1",
     "idb-keyval": "2.3.0",
     "locale-string": "1.0.0",
-    "moment": "^2.18.1",
+    "lodash.get": "^4.4.2",
```

Comment:
Trying to minimize any impact to the bundle size by just pulling in one helper function. I think `get` is really useful, especially with Flow's requirements that one not check properties on possibly undefined variables.